README.md

```diff
@@ -36,33 +32,67 @@ In particular, you can provide them with `--release`, `--dev`, or `--profile`
 flags to switch the compilation profile. If not option is provided, the scripts
 default to the `--release` profile.
 
-### Running application and examples
 For best experience, it is recommended to use the 
-`scripts/watch.sh --dev` in a second shell. In order to build the IDE application, 
-follow the steps below:
+`scripts/watch.sh --dev` in a second shell.
+
+### [Optional] Minimizing the WASM binary size
+After building the project you can use the `scripts/minimize_wasm.sh` to optimize 
+the binary and compress it by using `gzip`. After the script is complete, the
+final size is printed to stdout. Please note that in order to run the script, the
+[Binaryen](https://github.com/WebAssembly/binaryen) toolkit has to be installed
+on your system.
+
+### Web Application
+In order to build the IDE web application, follow the steps bellow:
 
 ```bash
-cd app
-npm install
-npm run start 
+ide/app$ npm install
```

Comment:
Don't we need to run the `build` script first to compile rust?

Reply:
I believe that after the changes I requested above will be introduced (titles to the sections), this should be obvious.